rules/sinks/storages/microsoft/caching/csharp.yaml

```diff
@@ -0,0 +1,18 @@
+sinks:
```

Comment:
why rule for local cache ?
cache makes sense only if the code runs on end-user devices, like JS or android apps

Reply:
yes right will remove them.